pkg/entities/guild_config_token.go

```diff
@@ -32,3 +32,20 @@ func (e *Entity) CreateGuildCustomTokenConfig(req request.UpsertCustomTokenConfi
 
 	return nil
 }
+
+func (e *Entity) ListAllTokenID(guildID string) ([]int, error) {
```

Comment:
join table `token` and `guild_config_token` -> query with guild_id 
not need to create 2 function like this